components/admin_console/license_settings/trial_license_card/trial_license_card.tsx

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import {FormattedMessage} from 'react-intl';
+
+import moment from 'moment';
+
+import {trackEvent} from 'actions/telemetry_actions';
+import {getBrowserTimezone} from 'utils/timezone';
+
+import PurchaseNowLink from 'components/announcement_bar/purchase_now_link/purchase_now_link';
+
+import './trial_license_card.scss';
+
+export interface Props {
+    license: any;
+}
+
+const TrialLicenseCard: React.FC<Props> = ({license}: Props) => {
+    const today = moment(Date.now());
+    const endOfLicense = moment(new Date(parseInt(license?.ExpiresAt, 10)));
+    const daysToEndLicense = endOfLicense.diff(today, 'days');
+    const hoursToEndLicense = endOfLicense.diff(today, 'hours');
+
+    const handleContactLinkClick = async (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        e.preventDefault();
+        trackEvent('admin', 'in_trial_contact_sales');
+        window.open('https://mattermost.com/contact-us/', '_blank');
+    };
+
+    const message = () => {
+        if (hoursToEndLicense > 24) {
+            return (
+                <FormattedMessage
+                    id='admin.license.trialCard.description'
+                    defaultMessage='Your free trial will expire in <b>{daysCount} {daysCount, plural, one {day} other {days}}</b>. Visit our customer portal to purchase a license now to continue using E10 & E20 features after trial ends.'
+                    values={{
+                        daysCount: daysToEndLicense,
+                        b: (chunk: any) => (<b>{chunk}</b>),
```

Comment:
Not a huge fan of the way we are bolding text. I think it's better to use `FormattedMarkdownMessage` and use `**` around the text you want bolded instead. Avoids this extra function passing.